template.go

```diff
@@ -38,59 +42,65 @@ type Engine struct {
 	Extension string
 	// layout variable name that incapsulates the template
 	LayoutName string
-	// determines if the engine parsed all templates
```

Comment:
we need to customize the version of the template core and mark it as breaking because we have removed the previously exposed properties

- [ ] increase the template core version with a major jump and change the pkg go.mods